test/base/src/jvmMain/kotlin/io/realm/test/platform/PlatformUtils.kt

```diff
@@ -35,7 +38,35 @@ actual object PlatformUtils {
 
     actual fun threadId(): ULong = Thread.currentThread().id.toULong()
 
+    @Suppress("ExplicitGarbageCollectionCall")
```

Comment:
Good point, we might do this if the `PlatformUtils` grows bigger, we can do a common JVM one and a specific per target since it is not possible now to implement partially some methods as you stated